src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -2276,6 +2331,21 @@ namespace EnergyPlus {
 		qdotRadZone_             = 0.0;
 	}
 
+	void
+	DCtoACInverter::setPVWattsDCCapacity(const Real64 dcCapacity)
+	{
+		if ( modelType_ != InverterModelType::pvWatts ) {
+			ShowFatalError("Setting the DC Capacity for the inverter only works with PVWatts Inverters.");
+		}
+		ratedPower_ = dcCapacity / pvWattsDCtoACSizeRatio_;
```

Comment:
I don't think we have ever fully resolved our pattern for private object variable naming.  The trailing underscore isn't terrible.  I think we should let this go in as one possible example and then as another class or two come along, make a decision based on those examples.

Reply:
Yeah, I went with the trailing underscore because that's what was used in the Electric Load Center Distribution class.